plugins/kubernetes/README.md

```diff
@@ -14,7 +14,12 @@ It is only meant for local development, and the setup for it can be found inside
 
 ## Surfacing your Kubernetes components as part of an entity
 
-### Adding the entity annotation
+There are 2 ways to surface your kubernetes components as part of an entity.
+The label selector takes precedence over the annotation/service id.
```

Comment:
in that case it should probably be at the top

Reply:
reorganized accordingly